src/benchmark/trulens/benchmark/benchmark_frameworks/tru_benchmark_experiment.py

```diff
@@ -71,9 +71,6 @@ def __init__(
             for agg_func in agg_funcs
         ]
 
-    def load_beir_dataset(self, *args, **kwargs):
-        pass  # TODO
-
```

Comment:
this will go into its own util class in the next PR for api changes: https://github.com/truera/trulens/pull/1353